lib/rubocop/cop/mixin/forbidden_name.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # This module encapsulates the ability to forbid certain identifiers or patterns.
+    module ForbiddenName
```

Comment:
I think this should probably be 2 mixins to match the Allowed* mixins (and to give more control the the users of the mixins)

Reply:
@bbatsov Done.